docs/egress.md

```diff
@@ -41,6 +41,10 @@ You may be interested in using this capability if any of the following apply:
 - You want to force outgoing external connections to leave the cluster via
   certain Nodes, for security controls, or due to network topology restrictions.
 
+**Note**: A dummy interface is automatically created on the Node hosting the
```

Comment:
It should mention the interface name, otherwise it's unclear which one it's talking about